prereqs.md

```diff
@@ -0,0 +1,7 @@
+[comment]: <> (list up any scenario-specific prerequirements the user needs to have installed, to guarantee a successful deployment)
```

Comment:
Why do we need an extra prerequisites file when the prerequisites are also mentioned in the README? Is there an feature in the TDD site that uses this file?